WORKSPACE.bazel

```diff
@@ -0,0 +1,5 @@
+workspace(name = "bazel_features")
+
+load(":deps.bzl", "bazel_features_deps")
+bazel_features_deps()
+
```

Comment:
Are there actual empty lines at the end? I could swear GitHub did not render full lines for the expected trailing newline character.

Reply:
We could just add Keith's buildifier presubmit to this repo and never talk about formatting again :)

Reply:
I ran buildifier over everything for now. will add the template stuff in a follow-up!